src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,6 +35,8 @@ public class RobotContainer {
    * awful.
    */
   public static Drive drive = new Drive();
+  public static Pneumatics pneumatics = new Pneumatics();
```

Comment:
JavaDoc all public fields (don't worry about anything you didn't create—that's out of scope for your branch and was already fixed by Mars and I)